whud_nav_client:
  map_frame_id: "map"
  body_frame_id: "nav_link"

whud_switch:
  state_machine_switch: false